src/app/components/SkipToContentWrapper.spec.tsx

```diff
@@ -26,15 +26,15 @@ describe('SkipToContentWrapper', () => {
 
     it('fails when main is not wrapped in a SkipToContentWrapper', () => {
         const breaks = () => renderToDom(<MessageProvider>
-            <MainContent/>
+            <MainContent isGenericStyle={true}/>
```

Comment:
why even have a prop for this if it is always `true`?

Reply:
It is sometimes false. When the `config.json` contains an entry for a book style to use then the value is false.